crates/sui-types/src/crypto.rs

```diff
@@ -287,28 +287,6 @@ impl EncodeDecodeBase64 for PublicKey {
     }
 }
 
-impl Serialize for PublicKey {
```

Comment:
doesn't it impact also non multisig accounts?

Reply:
discussed offline that enum PublicKey serde is only used for multisig.